packages/esm-home-app/src/dashboard-container/dashboard-container.scss

```diff
@@ -27,6 +27,36 @@
   }
 }
 
+// Overriding styles for RTL support
+html[dir='rtl'] {
+  .homePageWrapper {
+    flex-direction: row-reverse;
+  }
+
+  .dashboardContainer {
+    &.hasLeftNav {
+      // For RTL languages, use right margin instead of left margin
+      margin-left: 0;
+      margin-right: 16rem;
+    }
+  }
+
+  // Ensure the left navigation content flows properly in RTL
+  :global(.cds--side-nav) {
+    direction: rtl;
+  }
+
+  :global(.cds--side-nav__link) {
+    text-align: right;
+  }
+
+  // Handle active link borders for RTL
+  :global(.active-left-nav-link) {
+    border-left: none !important;
+    border-right: 0.125rem solid var(--brand-01) !important;
+  }
+}
```

Comment:
We can replace all of this by changing line 24 to:

```scss
  &.hasLeftNav {
    // Make space for the left nav menu
    margin-inline-start: 16rem;
  }
```

`margin-inline-start` should be able to account for both LTR and RTL locales and is a best practice for modern CSS. The left nav link style overrides should happen in esm-core, not here.